querydsl-jpa/src/main/java/com/querydsl/jpa/JPQLTemplates.java

```diff
@@ -106,8 +106,8 @@ protected JPQLTemplates(char escape, QueryHandler queryHandler) {
         add(Ops.MATCHES, "{0} like {1}  escape '" + escape + "'"); // TODO : support real regexes
         add(Ops.MATCHES_IC, "{0} like {1} escape '" + escape + "'"); // TODO : support real regexes
         add(Ops.LOWER, "lower({0})");
-        add(Ops.SUBSTR_1ARG, "substring({0},{1+'1's})", Precedence.ARITH_LOW);
-        add(Ops.SUBSTR_2ARGS, "substring({0},{1+'1's},{2-1s})", Precedence.ARITH_LOW);
+        add(Ops.SUBSTR_1ARG, "substring({0},{1+'1's})");
+        add(Ops.SUBSTR_2ARGS, "substring({0},{1+'1's},{2-1s})");
```

Comment:
Maybe put this in a separate commit for documentation purposes instead of `Add test`?